src/renderer/src/pages/game-details/hero/hero-panel-actions.tsx

```diff
@@ -43,23 +43,22 @@ export function HeroPanelActions({
   const { t } = useTranslation("game_details");
 
   const selectGameExecutable = async () => {
-    return window.electron
-      .showOpenDialog({
-        properties: ["openFile"],
-        filters: [
-          {
-            name: "Game executable",
-            extensions: window.electron.platform === "win32" ? ["exe"] : [],
```

Comment:
having an empty list on the filter actually makes the file selection show every possible file. this is not the cause of the bug

Reply:
you can test it. Run the app with the "exe" filter, and then run it again with the empty list

Reply:
I did tested and it's not working. At least on Linux `[]` means no files at all. When I change to `['*']` it goes like a charm. There is a mention about it in the docs ([see](https://www.electronjs.org/docs/latest/api/dialog#methods)), maybe you want to checkout.

I even record a video, just in case.

https://github.com/hydralauncher/hydra/assets/22862212/5fadcc98-b35d-4d9a-830c-156e3bcb8e12

Reply:
Note that the video was record using the code from `main`.

Reply:
Btw, the video also shows that #240 and #141 got fixed.

Reply:
Oh, I see. So I think a better modification would to set "*" for non-windows systems, and keep ".exe" for windows

Reply:
Have a "All Files (*)" can be usefull for cases like #302. What do you think?

Reply:
I see your point, but for now I'd rather we keep only the .exe for Windows and think better how to solve #302 on another PR

Reply:
Ok. I changed it.

```typescript
    const { canceled, filePaths } = await window.electron.showOpenDialog({
      title: t("game_executable_selection.title"),
      properties: ["openFile"],
      filters: [
        {
          name: window.electron.platform === "win32" ? t("game_executable_selection.executable_files") : t("game_executable_selection.all_files"),
          extensions: window.electron.platform === "win32" ? ["exe"] : ["*"],
        },
      ],
    });
```